### DISCUSSION TITLE
He throws to the top with every change

### DISCUSSION THREAD

**Discussion by miracim** [vercel/next.js#57567] [Help]
### Summary

Friends I am using NextJS 13.5 and I have a problem. I am making a page with Tailwindcss and now when the scroll comes, I go down and so on. But the problem is that every time I change the page throws me to the top. Whether this is footer or something else. I'm annoyed that every change throws me to the top of the page, how can I solve it, enough is enough!

**Comment by icyJoseph** [comment]
You mean on page transition, it scrolls to the top? I think you can use scroll false on next/link:

- https://nextjs.org/docs/app/api-reference/components/link#scroll

> Defaults to true. The default behavior of <Link> is to scroll to the top of a new route or to maintain the scroll position for backwards and forwards navigation. When false, next/link will not scroll to the top of the page after a navigation.
